src/main/java/org/codeforamerica/messaging/models/TemplateVariant.java

```diff
@@ -41,6 +45,11 @@ public class TemplateVariant {
     @JsonIgnore
     @ToString.Exclude
     Template template;
+    @OneToMany(mappedBy = "templateVariant", fetch = FetchType.EAGER)
```

Comment:
Without the eager fetch, the tests that use the `messages` list throw this error:
```
org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: org.codeforamerica.messaging.models.TemplateVariant.messages: could not initialize proxy - no Session
```
There is probably a way to get the test to initialize the list, but I don't know what it is.

Reply:
This may be a better solution - https://github.com/codeforamerica/messaging/pull/30/files#diff-861a3a31a3b50ffa921490a2347440a3ab627c871595bc9eca5e4bba43e4c8b4R11

Reply:
I found that using `@Transactional` will init the data. I'm wary of using it too much, but the test now pass and some basic manual tests are working too.

Reply:
Moving from EAGER to LAZY will reduce the incidence but will still be prohibitively expensive when called.